c2corg_ui/static/js/documentediting.js

```diff
@@ -488,6 +488,9 @@ app.DocumentEditingController.prototype.hasMissingProps = function(doc, showErro
       hasError = (!doc['lonlat'] || (doc['lonlat'][field] === null || doc['lonlat'][field] === undefined));
     } else if (field === 'date_start') {
       hasError = (doc['date_start'] === null || doc['date_start']  === undefined);
+    } else if (field === 'elevation' && doc['waypoint_type'] === 'climbing_indoor') {
```

Comment:
Don't you need to test that the current document is a waypoint (``type === 'waypoints'``)?